flow-typed/most.js

```diff
@@ -0,0 +1,414 @@
+declare module "most" {
+
+  declare type SeedValue<S, V> = { seed: S, value: V };
+  declare type TimeValue<V> = { time: number, value: V };
+
+  declare interface Generator<A, B, C> {}
+  declare interface Iterable<A> {}
```

Comment:
These are predefined in flow: https://github.com/facebook/flow/blob/master/lib/core.js#L484